project_milestone/README.rst

```diff
@@ -0,0 +1,83 @@
+=========================
+Project Milestone
+=========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/12.0/project_milestone
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-12-0/project-12-0-project_milestone
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/140/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module provides a project milestones on projects.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Have Manager rights for Project group to edit projects.
+#. On the project form, set the "Use Milestones" option to enable milestones for the project.
+#. Add the milestones to the project via the milestone tab on the project form.
+#. Tasks will show a "Milestone" where you can set tasks into the proper milestone.
```

Comment:
This is true and might be a road map improvement but I think it would need to be an option to enable/disable. Our company uses targets as more of a 'goal' than a hard set date. Our current project management teams use this target date as an estimated date, and then at the end of the project, to measure when and why they might miss them. After project completion, they evaluate the tasks and if the tasks go past the milestone target date, they use this information if it's common to adjust the templates for more accurate planning going forward. Floating or locked target dates didn't help, at least in our companies processes.